utreexo-validation-bip.md

```diff
@@ -138,14 +138,14 @@ as all proofs are with respect to the merkle roots of the entire set.
 For this reason, we define which UTXOs are not inserted to the accumulator.  Any 
 variations here will result in Utreexo nodes with incompatible proofs.
 
-##### Provably unspendable transaction outputs
+## Provably unspendable transaction outputs
```

Comment:
This section and the one below it should be inside the previous one, no? Like three `#` since this lists the outputs that are excluded.